sycl/include/sycl/accessor.hpp

```diff
@@ -2737,6 +2754,20 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(local_accessor) local_accessor
     return Property();
 #endif
   }
+
+  template <int Dims = Dimensions, typename = detail::enable_if_t<
+                                       !std::is_const_v<DataT> && Dims == 0>>
```

Comment:
Why are we not using `IsAccessReadOnly` as it is done for `accessor` class?

Reply:
I don't think `local_accessor` can be read-only because it would be impossible to use it.